systemtest/src/main/java/io/strimzi/systemtest/parallel/SuiteThreadController.java

```diff
@@ -26,29 +28,42 @@
 
     private static final long STARTING_DELAY = Duration.ofSeconds(5).toMillis();
     private static final Logger LOGGER = LogManager.getLogger(SuiteThreadController.class);
+    private static final String JUNIT_PARALLEL_COUNT_PROPERTY_NAME = "junit.jupiter.execution.parallel.config.fixed.parallelism";
 
     private static SuiteThreadController instance;
-
-    private AtomicInteger runningTestSuitesInParallelCount;
-    private AtomicBoolean isOpen;
+    private static List<String> waitingTestSuites;
+    private static Integer maxTestSuitesInParallel;
+    private static AtomicInteger runningTestSuitesInParallelCount;
+    private static AtomicBoolean isOpen;
+    private static AtomicBoolean isParallelSuiteReleased;
 
     public synchronized static SuiteThreadController getInstance() {
         if (instance == null) {
             instance = new SuiteThreadController();
+            if (System.getProperties().get(JUNIT_PARALLEL_COUNT_PROPERTY_NAME) != null) {
+                maxTestSuitesInParallel =  Integer.parseInt((String) System.getProperties().get(JUNIT_PARALLEL_COUNT_PROPERTY_NAME));
+            } else {
+                LOGGER.error("User did not specify junit.jupiter.execution.parallel.config.fixed.parallelism " +
```

Comment:
```suggestion
                LOGGER.warn("User did not specify junit.jupiter.execution.parallel.config.fixed.parallelism " +
```

or maybe even info si fine?

Reply:
I will set to `warn` :)